schemas/preconversion_assessment_schema_1.0.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Having the subschema specified (same as convert2rhel does) `"$schema": "https://json-schema.org/draft/2020-12/schema",` is causing deprecation warnings in pytest, do we need it? It doesn't seem to affect the validation

Reply:
I think if that doesn't affect the validation, then it is fine to remove.